docs/tactics.md

```diff
@@ -1337,3 +1337,20 @@ See also additional documentation of `using_well_founded` in
 * `@[simps]` reduces let-expressions where necessary.
 * If one of the fields is a partially applied constructor, we will eta-expand it
   (this likely never happens).
+
+### mk_simp_attribute
+
+The command `mk_simp_attribute simp_name "description"` creates a simp set with name `simp_name`.
+Lemmas tagged with `@[simp_name]` will be included when `simp with simp_name` is called.
+`mk_simp_attribute simp_name none` will use a default description.
+
+Appending the command with `with attr1 attr2 ...` will include all declarations tagged with
+`attr1`, `attr2`, ... in the new simp set.
+
+This command is preferred to using ``run_cmd mk_simp_aset `simp_name`` since it adds a doc string
```

Comment:
Should this be ``run_cmd mk_simp_attr `simp_name``?
```suggestion
This command is preferred to using ``run_cmd mk_simp_attr `simp_name`` since it adds a doc string
```

Reply:
Oops, yep.